src/driver/driver_api.cc

```diff
@@ -569,15 +569,14 @@ transform::Sequential MixedModulePassManager(IRModule mixed_mod, Target target)
 
   Array<Pass> mixed_pass_list;
 
-  mixed_pass_list.push_back(tir::transform::FP8ComputeLegalize(target));
+  mixed_pass_list.push_back(tir::transform::BindTarget(target));
```

Comment:
Can we add a comment here specifying that the `BindTarget` should occur first in this sequence, so that later passes can rely on the target attribute being present?  (It looks like both `VerifyVTCMLimit` and `LowerVtcmAlloc` were implemented more recently than `BindTarget`, and probably should have been placed after `BindTarget` at that point.)

Reply:
Yeah will do

Reply:
>
> but also, in principle, allows for PrimFuncs with different targets to coexist and be handled correctly by the passes.

Here BindTarget is applying the target to all functions in the mixed IRModule, no?

Reply:
Yeah that's what it does. In principle, we could set the attributes differently.

Reply:
Slight nitpick: All functions that do not already have the target attribute.  A function could already have the target attribute, such as defining a module that contains kernels for multiple different targets.